src/apps/search-result/search-result.tsx

```diff
@@ -164,22 +158,23 @@ const SearchResult: React.FC<SearchResultProps> = ({ q, pageSize }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [campaignData]);
 
+  // This is a hack to clear filters when the user navigates
+  // to the search page from links where there should be no filters.
+  useEffect(() => {
+    const filtersUrlParam = getUrlQueryParam("filters");
+    if (filtersUrlParam !== "withFilters") clearFilter();
```

Comment:
Isn't it possible to maintain the filters so if the user goes back in history they are still intact the moment "withFilters" is in the URL?
With that model it is a matter about to ignore/use the stored filters depending on the "withFilters" param.
Does it makes sense?

Reply:
If we stick to that model. This `useEffect` would not be needed.

Reply:
Actually I think it could be solved with just returning empty filters from the filterHandler if "withFilters" is not in the URL